misk-redis/src/main/kotlin/misk/redis/JedisPoolWithMetrics.kt

```diff
@@ -1,78 +0,0 @@
-package misk.redis
-
-import org.apache.commons.pool2.PooledObject
-import redis.clients.jedis.Jedis
-import redis.clients.jedis.JedisFactory
-import redis.clients.jedis.JedisPool
-import redis.clients.jedis.JedisPoolConfig
-import redis.clients.jedis.Protocol
-
-internal class JedisPoolWithMetrics(
```

Comment:
I'm working on a PR to add bucket4j support to misk, as an alternative to `cash-permit`. Unfortunately, its redis backend implementation currently requires a `JedisCluster` or a `Pool<Jedis>`. I can work on a PR to the upstream that adds support for `UnifiedJedis`, but until then could we keep `JedisPoolWithMetrics`? I'll just instantiate a dedicated pool in the bucket4j redis module

Reply:
I will be adding the `JedisCluster` support next. But yes, I can revert `JedisPoolWithMetrics` for now and clean it up on a later PR. Also I just realized it was deleted here, I just renamed it.